### ISSUE TITLE
color coding without text - redesign not meet WCAG?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag-redesign#39]
Hey @michael-n-cooper and @iadawn , In researching an [issue we received](https://github.com/w3c/wcag/issues/2849) on potentially WCAG docs not meeting WCAG requirements, I looked at the redesigned pages. Unfortunately the problem is potentially even worse.

Current design has color for Notes  and Examples with "NOTE" in text and no such text for examples:
<img src="https://user-images.githubusercontent.com/6991814/207764759-b3633555-4a87-4356-86fc-d2ab3ee5553e.png" width="357">

The redesign has boxes around the notes, but "Note" is missing. And the examples have nothing. sigh.
<img src="https://user-images.githubusercontent.com/6991814/207767731-4bfd3dbb-98de-4850-b0d6-a8237c62040a.png" width="567">

**Comment by michael-n-cooper** [comment]
Discussion with @iadawn will add "Note" and "Example" header text.

**Comment by shawna-slh** [comment]
+1 to "Note" and "Example" not all caps
-1 to header text (I think inline is better, at least for the examples)
(and related: -1 to the visual design with tons of padding)
however, those aren't essential, so I put them in another issue with lower priority https://github.com/w3c/wcag-redesign/issues/40

"Note" is now showing up in the preview, and "Example" is probably the same bug as https://github.com/w3c/wcag/issues/2849

So, I'm closing this issue. Feel free to re-open if I missed something.
